nipype/interfaces/bids_utils.py

```diff
@@ -6,15 +6,9 @@
 
 BIDSDataGrabber: Query data from BIDS dataset using pybids grabbids.
 
-Change directory to provide relative paths for doctests
->>> import os
->>> import bids
->>> filepath = os.path.realpath(os.path.dirname(bids.__file__))
->>> datadir = os.path.realpath(os.path.join(filepath, 'grabbids/tests/data/'))
->>> os.chdir(datadir)
```

Comment:
you can replace this with the standard header that changes to the testing/data folder.

Reply:
Aren't we skipping these tests anyways since it would require pybids to be installed?

Reply:
correct, but we can make a directory `ds005` with a dummy file (since git does not store empty directories), just not call the bidsgrabber run function. we won't actually call pybids here.